connector/connector.go

```diff
@@ -57,11 +57,7 @@ type Connector interface {
 	// GetUpdates returns a stream of updates that the gluon server should apply.
 	GetUpdates() <-chan imap.Update
 
-	// Pause pauses the stream of updates.
-	Pause()
-
-	// Resume resumes the stream of updates.
-	Resume()
+	Poll()
```

Comment:
Can you add a comment on what `Poll()` is expected to do and how it interacts with the imap.Update channel from `GetUpdates()`